### DISCUSSION TITLE
CDN version to use in the <script> tag

### DISCUSSION THREAD

**Discussion by Mashiane** [bcakmakoglu/vue-flow#64] [Q&A] [answered]
Hi there

Is it possible to have a dist for this that one can use in the <script> tag?

Thank you so much!

**Comment by bcakmakoglu** [comment, marked as answer]
Currently, no.
It might be something I add in the future though, so I will keep this thread open as an interest check. 😄

**Reply by bcakmakoglu** [reply]
@Mashiane 
I have to correct myself here.

You can include it from CDN:
`<script src="https://cdn.jsdelivr.net/npm/@braks/vue-flow"></script>`

**Reply by agtabesh** [reply]
Thank you so much for such an exciting library. well done.
Could you please release the last version in `jsdelivr.net` periodically?
In most cases, we must load it into the page directly using the `<script>` tag.

Regards

**Reply by bcakmakoglu** [reply]
I don't release anything myself to jsDelivr, it's just done automatically: https://www.jsdelivr.com/package/npm/@vue-flow/core

You can get the latest version here.

**Reply by agtabesh** [reply]
It didn't load because of my internet connection! Now it works.

But this code does not work. Nothing is shown on the page.
could you please check this out to find out whether it relates to packaging?

```HTML
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <script src="https://unpkg.com/vue@3/dist/vue.global.prod.js"></script>
  <script src="https://cdn.jsdelivr.net/npm/@vue-flow/core@1.15.1/dist/vue-flow-core.iife.min.js"></script>
  <link href="https://cdn.jsdelivr.net/npm/@vue-flow/core@1.15.1/dist/style.css" rel="stylesheet">
  <link href="https://cdn.jsdelivr.net/npm/@vue-flow/core@1.15.1/dist/theme-default.min.css" rel="stylesheet">
</head>
<body>
    <div id="app">
      <h1>{{ count }}</h1>
      <button @click="increment">Increment</button>
      <VueFlow v-model="elements" />
    </div>

    <script>
      const { ref, createApp } = Vue
      createApp({
        setup() {
          const count = ref(0)
          const elements = ref([
              { id: '1', type: 'input', label: 'Node 1', position: { x: 250, y: 5 } },
              { id: '2', label: 'Node 2', position: { x: 100, y: 100 }, },
              { id: '3', label: 'Node 3', position: { x: 400, y: 100 } },
              { id: '4', type: 'output', label: 'Node 4', position: { x: 400, y: 200 } },
              { id: 'e1-3', source: '1', target: '3' },
              { id: 'e1-2', source: '1', target: '2', animated: true }
            ])
          const increment = () => {
            count.value++
          }

          return {
            count,
            elements,
            increment
          }
        }
      }).mount('#app')
    </script>
</body>
</html>

```

**Reply by bcakmakoglu** [reply]
You need to pass a proper height/width to your VueFlow component
Try wrapping it in a div like `<div style="height: 100vh; width: 100vw">` or something

**Reply by agtabesh** [reply]
I tried but it didn't work either
```html
<div style="height: 100vh; width: 100vw">
  <VueFlow v-model="elements" />
</div>
```

**Reply by bcakmakoglu** [reply]
Any errors you got?

**Reply by agtabesh** [reply]
Nothing related!

**Reply by bcakmakoglu** [reply]
Mh... No idea why the App doesn't actually mount the component properly.
This works:
```jsx
<div id="app" style="height: 100vh"></div>

<script>
  const { ref, createApp, h } = Vue
  const { VueFlow } = vueFlowCore

  const app = createApp({
    setup() {
      const elements = ref([
        { id: '1', type: 'input', label: 'Node 1', position: { x: 250, y: 5 } },
        { id: '2', label: 'Node 2', position: { x: 100, y: 100 } },
        { id: '3', label: 'Node 3', position: { x: 400, y: 100 } },
        { id: '4', type: 'output', label: 'Node 4', position: { x: 400, y: 200 } },
        { id: 'e1-3', source: '1', target: '3' },
        { id: 'e1-2', source: '1', target: '2', animated: true },
      ])

      return () => h(VueFlow, { modelValue: elements.value })
    },
  })

  process = {
    env: {
      NODE_ENV: 'development',
    },
  }

  app.mount('#app')
</script>
```

Though there seems to be an issue with the library looking for `process.env` which doesn't work from this package (hence I set that global myself, which is not good practice obviously).

**Reply by agtabesh** [reply]
Yes! I think the problem comes from there.
By the way, thanks a lot.

**Reply by bcakmakoglu** [reply]
Though other components like Background, Minimap don't seem to work, I'll take a look at that and try to fix the process.env issue as well :)

**Reply by agtabesh** [reply]
Very well though, I'll be happy if there is anything I can help with.
Thanks again.

**Comment by Mashiane** [comment]
Awesome, thank you so much, will check it out. Your work is amazing, congratulations.
